README.md

```diff
@@ -765,4 +758,76 @@ EXAMPLE
   }
 ```
 
- 
\ No newline at end of file
+## `sfpowerkit:profile:sync`
+
+retrieve profiles from the salesforce org with all their configurations. Run this command preferably again the production org. Use the merge equivalent again other type of orgs.
```

Comment:
Can you make it clear what this command does?I believe for this command to be run you should have the full set of metadata

Reply:
No need to have the full set of metadata. The command read the metadata using the `metadata.readSync` function. Then complete userPermissions for permissions not retrieved (Those unchecked in the org). The list of supported permissions is computed using a Describe on the PermissionSet Sobject. Then the command complete ObjectPermission for objects with No Read Access. The list of supported objects is computed by running the query `SELECT SobjectType, count(Id) From ObjectPermissions Group By sObjectType`
For standard profiles, the List of UserPermissions is removed as salesforce does not allow to edit user permissions for standard Profiles. So no need to retrieve them.

Reply:
Lets elaborate the readme to make it bit more clear